idtype/idtype.go

```diff
@@ -71,6 +71,8 @@ var (
 	PayoutEvent               Type = 0x1FB // record of payout of failure to payout
 	PayoutProfile             Type = 0x1FC // Payout profile for providers
 
+	ActivityEvent Type = 0x25D
```

Comment:
Spacing.

Reply:
That's the default from gofmt since it's in a new line it won't align with the rest